app/workers/my_anime_list_list_worker.rb

```diff
@@ -1,19 +1,45 @@
 class MyAnimeListListWorker
   include Sidekiq::Worker
 
-  def perform(user_id)
-    kitsu_library(user_id).each do |library_entry_id|
+  def perform(linked_account_id, user_id)
+    kitsu_library(user_id).each do |library_entry|
+      # create the logs here before the next worker
+      library_entry_log = create_log(library_entry, linked_account_id)
+
       MyAnimeListSyncWorker.perform_async(
-        library_entry_id: library_entry_id,
-        method: 'create/update'
+        library_entry_id: library_entry.id,
+        # HACK: I am unable to determine the method
+        # through the initial sync because I don't access their
+        # mal list. For now everything will be a create action,
+        # but at some point we can update to get the correct action.
+        # Same issue with the LibraryEntryLog performed_action
+        method: 'create',
```

Comment:
There is not, so I was thinking what we could do is pass in the method as `'upsert'` and then inside the `execute_method` method I can just update the `method=` setter to be specifically `update` or `create`. Would something like this work?